### PR TITLE
https://github.com/Bitwise-01/Instagram-/pull/199#issue-556245165

### PR DESCRIPTION
**Pull request by aqwsoo**

### CODE REVIEW

**Review by iidiid — COMMENTED** [review]
.

**Comment by shyamkuntal** [comment]
it takes a very lot of time to crack

File: https:/instagram.com/mana.l2?igshid=1967jrh1ixhbu [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (dhhfjfii) [inline https:/instagram.com/mana.l2?igshid=1967jrh1ixhbu:1]: https://instagram.com/karlahernandez241208?utm_medium=copy_link

Comment (iamcccn) [reply https:/instagram.com/mana.l2?igshid=1967jrh1ixhbu:1]: Follow back

Comment (dhhfjfii) [reply https:/instagram.com/mana.l2?igshid=1967jrh1ixhbu:1]: instagram.com/mana.l2?igshid=1967jrh1ixhbu

**Review by dhhfjfii — APPROVED** [review]

File: https:/instagram.com/mana.l2?igshid=1967jrh1ixhbu (hunk 1 above)
Comment (iamcccn) [reply https:/instagram.com/mana.l2?igshid=1967jrh1ixhbu:1]: Http://www.instagram.com/iam_cccn

**Pull request closed without merging**